cli/test/e2e/package_repository_test.go

```diff
@@ -66,8 +63,7 @@ func TestPackageRepository(t *testing.T) {
 
 	logger.Section("kicking a repository", func() {
 		out := kappCtrl.Run([]string{"package", "repository", "kick", "-r", pkgrName})
-		require.Contains(t, out, "Fetch succeeded")
-		require.Contains(t, out, "Template succeeded")
+
 		require.Contains(t, out, "Deploy succeeded")
```

Comment:
Could check last reconciled here instead. But not a fan of time based tests in general as they tend to flake up 🤔

Reply:
> Could check last reconciled here instead.

Yeah, that's what I was thinking.

> But not a fan of time based tests in general as they tend to flake up 🤔

I am not sure what you mean by time based tests here.